IBERTpy/python/generate_all_plots.py

```diff
@@ -13,7 +13,7 @@
 minlog10ber = -8
 overwrite = True
 
-filename_i_list = glob('../scans/CM01/fpga1/05-21-21/*.csv') # ---------------> need to be modified as a variable input
+filename_i_list = glob('../scans/CM203/06-08-22/*.csv') # ---------------> date needs to be modified here as a variable input before running (e.g. 06-13-22)
```

Comment:
can you make the argument to the glob here configurable w/o editing the script

Reply:
I made this file work without needing to edit the script, but I only allowed for the user to change the board id and date in the file path from the command line.  All of these files require outputs to be saved in a particular way in order to function.  Although, now that we have the new /nfs/cms/tracktrigger directory, perhaps I could rewrite the files so that all or some of the outputs are saved there.  I could also rewrite the files so that the outputs are saved in both locations.  Do we just want the eyescan pdfs in the /nfs/cms/tracktrigger directory or everything including the eyescan and autotuning csv files?

Reply:
all data outputs should be saved somewhere in /nfs/cms/tracktrigger.

Reply:
I reconfigured the files to save to both locations: the output directories for Cornell_CM_Production_Scripts and /nfs/cms/tracktrigger.